skimage/transform/_geometric.py

```diff
@@ -257,6 +257,58 @@ class FundamentalMatrixTransform(GeometricTransform):
     params : (3, 3) array
         Fundamental matrix.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> import skimage as ski
+    >>> tform_matrix = ski.transform.FundamentalMatrixTransform()
```

Comment:
```suggestion
    >>> tform_matrix = FundamentalMatrixTransform()
```
(iirc this explicit import is unnecessary)

Reply:
While it's not necessary for the tests to pass I think complete imports should be part of each doctests. That makes them easier to copy and also more explicit in a few ways. Also I recall that in some edge cases doctest get's confused if functions or classes share a name of a submodule. Using `ski.module.some_object` avoids this.

It's also more consistent since we have to include `import skimage as ski` in doctests that uses API from other modules.

Reply:
@lagru ok, I get your rationale; I agree it's more robust and more extensible to include `import skimage as ski` in all doctests. Taking note.

@ana42742 please revert my suggestion; sorry for the noise!